README.md

```diff
@@ -1 +1,208 @@
-# hello_microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+  
+  <br/>
+
+  <h3><b>Hello Microverse!</b></h3>
+
+</div>
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+
+# 📖 [Hello Microverse!] <a name="about-project"></a>
+
+> This is my first project at Microverse. The aim of this project is to learn how to make a professional pull request and how to use GitHub properly in all my future projects.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+- Git
+- Node.js
+- Linters
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
```

Comment:
[OPTIONAL] - Kindly consider removing all the section descriptions. 

e.g "> Describe the tech stack and include only the relevant sections that apply to your project." 

They are there just as a guide for customizing the readme. so we should remove them after adding our custom content 🤝